src/redux/books/books.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Books = () => (
+  <div className="main-container">
+    <div className="wrap">
+      <div className="card">
+        <div className="card-header">
+          <div className="title">
+            <div className="row">
+              <div className="col">
+                Bookstore CMS
+              </div>
+              <div className="col">
+                <Link className="links" to="/">BOOKS</Link>
+              </div>
+              <div className="col">
+                <Link className="links" to="/categories">CATEGORIES</Link>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="card-body">
+          <div className="card">
+            <div className="card-body">
+              <div className="details">
+                <div className="row">
```

Comment:
- [ ] The **`book.js`** should be a **`propTypes`** for the **Title** and **Author** see the screenshot 👇

![hfhf](https://user-images.githubusercontent.com/61562626/131416151-c4a1c5da-49cf-48ed-be5b-15650a4ad899.jpeg)